### PR TITLE
Updates the Talos

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
talos_2

Updates the Talos. 

- added IRMG space suits
- replaced pump shotguns with Mastiffs and small magazines
- reduced burn chamber to 1x3 tiles
- reduced Recruit slots to 5
- increased Corpsman slots to 2

## Changelog

:cl:
tweak: Updated Talos with new assets, adjusted engine and job slots
/:cl:

### CODE REVIEW

File: _maps/configs/inteq_talos.json [hunk 1]
```diff
@@ -28,15 +28,15 @@
...
+			"slots": 2
 		},
 		"Enforcer": {
 			"outfit": "/datum/outfit/job/security/inteq",
 			"slots": 1
 		},
 		"Recruit": {
 			"outfit": "/datum/outfit/job/assistant/inteq",
-			"slots": 7
+			"slots": 4
```
Comment (Erikafox) [inline _maps/configs/inteq_talos.json:39]: I would honestly consider adding another enforcer and knocking this down to 3 or even 2.

**Review by Erikafox — APPROVED** [review]

File: _maps/configs/inteq_talos.json (hunk 1 above)
Comment (Apogee-dev) [reply _maps/configs/inteq_talos.json:39]: Hard no on the second enforcer- It's a support ship, after all, and I want to keep the combat crew relatively scarce. We'll see how the recruit cap plays out in testing.

**Comment by Zevotech** [comment]
kill that rolling table or so help me god the next time my medkits get eaten by it

**Comment by Erikafox** [comment]
image
I want the window back
image

**Comment by Latentish** [comment]
Hm tm'able.

**Pull request merged**